upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -66,6 +66,10 @@ func buildCloudupTags(cluster *api.Cluster) (sets.String, error) {
 		{
 			tags.Insert("_aws")
 		}
+	case "digitalocean":
+		{
+			tags.Insert("_do")
```

Comment:
Hopefully we'll be rid of these tags soon, but I agree that for now we should add them - more logically coherent to be consistent with the other providers, doesn't create a lot more work